packages/test-helper/CHANGELOG.md

```diff
@@ -0,0 +1,11 @@
+# Change Log
+
+All notable changes to this project will be documented in this file.
+See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
+
+### 0.1.2 (2024-07-11)
```

Comment:
let's change the date when we releasing.

Reply:
removed the changelog since it will be automatically generated